package.json

```diff
@@ -28,7 +29,8 @@
     "react-native-web": "^0.9.4",
     "react-scripts": "2.0.5",
     "react-sizes": "^1.0.4",
-    "styled-components": "^4.0.3"
+    "styled-components": "^4.0.3",
+    "uuid": "^3.3.2"
```

Comment:
It's not an issue so no need to change anything in this PR, more a generic comment: as people will eventually have to pay for bandwidth when downloading from Swarm, the size of the apps codebase could become one of the metrics we need to watch, and be mindful about.
The `uuid` package is a good solution if we need actual standard UUIDs, but if the use case is to have a random ID, [nanoid](https://github.com/ai/nanoid) might be a better choice.